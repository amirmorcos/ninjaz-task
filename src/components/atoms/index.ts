import Icon from "./Icon/Icon";
import Divider from "./Divider/Divider";
import Avatar from "./Avatar/Avatar";

export { Icon, Divider, Avatar };
