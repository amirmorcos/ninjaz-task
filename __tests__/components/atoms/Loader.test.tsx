import React from "react";
import renderer from "react-test-renderer";
import { Loader } from "../../../src/components/atoms/index";

jest.mock("react-native-vector-icons/Ionicons", () => "Icon");
jest.mock("react-native-toast-message", () => ({
  show: jest.fn(),
  hide: jest.fn(),
}));
jest.mock("react-native-size-matters", () => ({
  scale: () => {},
  verticalScale: () => {},
  moderateScale: () => {},
}));

test("renders correctly", () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
