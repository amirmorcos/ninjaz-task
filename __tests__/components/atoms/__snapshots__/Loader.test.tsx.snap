// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <ActivityIndicator />
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <ActivityIndicator />
</View>
`;
